.circleci/config.yml

```diff
@@ -52,6 +64,24 @@ commands:
             echo 'export NVM_DIR="/opt/circleci/.nvm"' >> $BASH_ENV
             echo "[ -s \"$NVM_DIR/nvm.sh\" ] && . \"$NVM_DIR/nvm.sh\"" >> $BASH_ENV
 
+  run-owasp-scan:
```

Comment:
Allows simpler calling of the zap-scanner.sh script without repeating logic to store artifacts alongside running the script.